app-admin/logstash-bin: bump to 6.8.1/7.2.0

hydrapolic opened the pull request:
Thanks :)

mgorny commented on app-admin/logstash-bin/logstash-bin-6.8.1.ebuild:
```diff
@@ -0,0 +1,77 @@
+S="${WORKDIR}/${MY_P}"
+
+pkg_setup() {
+	enewgroup ${MY_PN}
```
Could you update it to use GLEP 81 user/group packages, and send RFC for UID/GID assignment? https://wiki.gentoo.org/wiki/Project:Quality_Assurance/UID_GID_Assignment is table of UIDs/GIDs already in use.

hydrapolic replied:
Sure, will do.

mgorny commented on acct-group/logstash/Manifest:
```diff
@@ -0,0 +1,2 @@
+EBUILD logstash-0.ebuild 201 BLAKE2B 3bd8012acd4f2fd399b140d0b4e0ee2f4d1465c5a140cff3296bcb5923c4abad28a4fc50502b55f04e1e8a5ccbceada9c1ae4de543c8f19f0484888feed3272f SHA512 5a3dfdfe0e17b7ca5723dd5027f7d3dc92669fd3ca138d8308ee29719a2711d0ddea985ebbba79d20a005482427f42be2199111fe2e1f7818ccb49afa9ef5730
```
Not like this ;-P.

mgorny commented on acct-group/logstash/logstash-0.ebuild:
```diff
@@ -0,0 +1,10 @@
+
+DESCRIPTION="Logstash program group"
+ACCT_GROUP_ID=270
+KEYWORDS="~amd64"
```
No KEYWORDS.

hydrapolic replied:
I saw it in mysql, removing.

mgorny commented on app-admin/logstash-bin/logstash-bin-6.8.1.ebuild:
```diff
@@ -0,0 +1,75 @@
+RESTRICT="strip"
+QA_PREBUILT="opt/logstash/vendor/jruby/lib/jni/*/libjffi*.so"
+
+
```
DoubleEmptyLine™

hydrapolic replied:
Removed :)

mgorny commented on acct-user/logstash/logstash-0.ebuild:
```diff
@@ -0,0 +1,14 @@
+ACCT_USER_HOME_PERMS=0770
+ACCT_USER_GROUPS=( logstash )
+acct-user_add_deps
+KEYWORDS="~amd64"
```
Same.

mgorny commented on acct-user/logstash/logstash-0.ebuild:
```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Gentoo Authors
```
This is out of order.

hydrapolic replied:
You mean the commit? It's strange, because locally this commit is the second one, hm..

hydrapolic replied:
After another rebase and force push it seems fine now.

hydrapolic commented:
@mgorny  I thought only devs can send to gentoo-dev. If you say it's ok I'll send a notice to gentoo-dev about the UID/GID 270 allocation.

hydrapolic commented:
I haven't found logstash user/gid in other databases (fedora, etc.)

juippis commented:
Every proxied maintainer was whitelisted (allowed) to post on gentoo-dev, because many proxy-maint tasks require that right. Unless you're explicitly banned... go for it :)

hydrapolic commented:
https://archives.gentoo.org/gentoo-dev/message/ee2cb65a29eb671bd2f6d1730ebc1883

hydrapolic commented:
TODO:
drop ACCT_USER_HOME_PERMS=0770
drop keepdir "/var/lib/${MY_PN}" from the ebuild

hydrapolic commented:
Updated per https://archives.gentoo.org/gentoo-dev/message/63d07af38f5fc37b9d84f19c7d6f2bd4

The perms on the home dir are set via the init script to 0750, so setting in the acct-user/logstash ebuild too. Removed duplicate keepdir from logstash ebuilds.

hydrapolic commented:
Thanks @mgorny @juippis
